### PR TITLE
Cleanshot update

### PR DESCRIPTION
**Pull request by Aayush9029**
## Description
Cleanshot X introduced a new all in one button! I'm updating the extension to support that feature.

> I added a custom icon, a new all-in-one.tsx file and updated package.json accordingly 

## Screencast
CleanShot 2022-03-24 at 22 14 51@2x

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `metadata` folder

### CODE REVIEW

File: extensions/cleanshotx/package.json [hunk 1]
```diff
@@ -1,136 +1,143 @@
...
+        {
+            "name": "pin",
+            "title": "Pin Screenshot",
+            "subtitle": "CleanShot X",
+            "description": "Pin Screenshot to the Screen.",
+            "icon": "pin.png",
+            "mode": "no-view"
+        }
+    ],
+    "dependencies": {
```
Comment (pernielsentikaer) [inline extensions/cleanshotx/package.json:122]:
```suggestion
    "categories": ["Applications"],
    "dependencies": {
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @Aayush9029 👋 

Thanks for creating the changelog, I can see that screenshots are almost impossible to make 😄  I added a suggestion, could you look into it?

**Comment by Aayush9029** [comment]
@pernielsentikaer Done. Let me know if there's anything that needs to be changed

**Review by pernielsentikaer — APPROVED** [review]
Hi @Aayush9029 👋 

Thanks for the fast push, approved 🚀

**Pull request merged**